Source/Core/Core/Debugger/RSO.cpp

```diff
@@ -380,14 +379,21 @@ void RSOView::Apply(PPCSymbolDB* symbol_db) const
       Symbol* symbol = symbol_db->AddFunction(address);
       if (!symbol)
         symbol = symbol_db->GetSymbolFromAddr(address);
+
+      const std::string export_name = GetExportName(rso_export);
       if (symbol)
       {
-        symbol->name = GetExportName(rso_export);
-        count += 1;
+        // Function symbol
+        symbol->name = export_name;
+      }
+      else
+      {
+        // Data symbol
+        symbol_db->AddKnownSymbol(address, 0, export_name, Symbol::Type::Data);
```

Comment:
The main reason is because there is no symbol size field, AFAIK. The goal of this PR is to show these symbols in the memory view (just like ELF reader does by adding data symbol).

In theory, there is a hash field for each (export) symbol that supposedly correspond to the hash of said symbol data. If that's true and I know how the hash is computed, I could try to use it in order to determine the size of each symbol. However, that's a bit overkill for the amount of symbol and just for displaying them on the memory view.